calibration.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3   
+import time 
+from robot import 
```

Comment:
add Robot to the end of this line

Reply:
@Bir9, can you please fix this line and use an instance of the robot in the `calibration` function?